packages/images/src/main.test.ts

```diff
@@ -0,0 +1,346 @@
+      })
+
+      test('matches on `/.netlify/images/', async () => {
+        const imageHandler = new ImageHandler({
+          logger: createMockLogger(),
+          originServerAddress: 'http://localhost:5173',
+        })
+
+        const url = new URL('/.netlify/images/', 'https://netlify.com')
+        url.searchParams.set('url', 'image.png')
+
+        const match = imageHandler.match(new Request(url))
+
+        expect(match).toBeDefined()
+
+        const response = await match!.handle()
+
+        expect(response.ok).toBe(true)
+        expect(response).toMatchObject(mockedIpxResponse)
+      })
+
+      test('does not match on `/.netlify/foo', () => {
+        const imageHandler = new ImageHandler({
+          logger: createMockLogger(),
+        })
+
+        const url = new URL('/.netlify/foo', 'https://netlify.com')
+        url.searchParams.set('url', 'image.png')
+
+        const match = imageHandler.match(new Request(url))
+
+        expect(match).not.toBeDefined()
+      })
+    })
+
+    describe('request methods', () => {
+      test('allows GET requests', async () => {
+        const imageHandler = new ImageHandler({
+          logger: createMockLogger(),
+          originServerAddress: 'http://localhost:5173',
+        })
+
+        const url = new URL('/.netlify/images', 'https://netlify.com')
+        url.searchParams.set('url', 'image.png')
+
+        const match = imageHandler.match(new Request(url))
+
+        expect(match).toBeDefined()
+
+        const response = await match!.handle()
+
+        expect(response.ok).toBe(true)
+        expect(response).toMatchObject(mockedIpxResponse)
+      })
+
+      test('does not allow POST requests', async () => {
+        const imageHandler = new ImageHandler({
+          logger: createMockLogger(),
+        })
+
+        const url = new URL('/.netlify/images', 'https://netlify.com')
+        url.searchParams.set('url', 'image.png')
+
+        const match = imageHandler.match(new Request(url, { method: 'POST' }))
+
+        expect(match).toBeDefined()
+
+        const response = await match!.handle()
+
+        expect(response.ok).toBe(false)
+        expect(await response.text()).toBe('Method Not Allowed')
+      })
+    })
+
+    describe('local images', () => {
+      let originServer: http.Server
+      let originServerAddress: string
+      const LOCAL_IMAGE_PATH = '/local/image.jpg'
+      const LOCAL_IMAGE_WIDTH = 800
+      const LOCAL_IMAGE_HEIGHT = 400
+
+      beforeAll(async () => {
+        ;[originServer, originServerAddress] = await new Promise<[http.Server, string]>((resolve, reject) => {
+          const originServer = http.createServer(function originHandler(req, res) {
+            if (req.url !== LOCAL_IMAGE_PATH) {
+              res.writeHead(404, { 'Content-Type': 'text/plain' })
+              res.end('Not Found')
+              return
+            }
+
+            generateImage(LOCAL_IMAGE_WIDTH, LOCAL_IMAGE_HEIGHT)
```

Comment:
I have some typescript issues showing up here, not sure why they aren't coming up in CI, but they show up when I run `npm run lint`

Reply:
Yes, those are the same issue I'm seeing. I agree, it's a lint setup issue not a problem within the code changes